client/src/pages/Bookings/Bookings.tsx

```diff
@@ -0,0 +1,107 @@
+import { useEffect } from 'react';
+import { useAuth } from '../../context/useAuthContext';
+import { useSocket } from '../../context/useSocketContext';
+import { useHistory } from 'react-router-dom';
+import { Box, Card, CircularProgress, Grid } from '@mui/material';
+import Calendar from '../../components/Calendar/Calendar';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import NextBooking from '../../components/NextBookings/NextBooking';
+import { Booking } from '../../interface/Booking';
+import BookingList from '../../components/BookingList/BookingList';
+
+const inMemoryNextBooking: Booking = {
+  start: new Date('2022-01-20T10:00'),
+  end: new Date('2022-01-20T12:00'),
+  status: 'accepted',
+  user: {
+    name: 'Norma Byes',
+    email: 'norma.byes@example.com',
+  },
+};
+
+const inMemoryCurrentBookings: Booking[] = [
+  {
+    user: {
+      name: 'Charles Compton',
+      email: 'charles.compton@example.com',
+    },
+    start: new Date('2022-01-20T19:00'),
+    end: new Date('2022-01-20T21:00'),
+    status: 'accepted',
+  },
+  {
+    user: {
+      name: 'Joan Blackeny',
+      email: 'joan.blackeny@example.com',
+    },
+    start: new Date('2022-01-23T08:00'),
+    end: new Date('2022-01-23T12:00'),
+    status: 'declined',
+  },
+];
+
+const inMemoryPastBookings: Booking[] = [
+  {
+    user: {
+      name: 'Michael Carnahan',
+      email: 'michael.carnahan@example.com',
+    },
+    start: new Date('2022-01-16T15:00'),
+    end: new Date('2022-01-16T22:00'),
+    status: 'accepted',
+  },
+];
+
+const combinedBookings = [...inMemoryCurrentBookings, ...inMemoryPastBookings, inMemoryNextBooking];
+const filteredBookings = combinedBookings.filter((b) => b.status !== 'declined');
+console.log(filteredBookings.map((b) => b.start));
```

Comment:
Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!